lib/package.rb

```diff
@@ -38,23 +38,31 @@ def self.depends_on (dependency = nil)
   end
 
   def self.get_url (architecture)
-    if !@build_from_source && @binary_url && @binary_url.has_key?(architecture)
+    if !@build_from_source and @binary_url and @binary_url.has_key?(architecture)
       return @binary_url[architecture]
     else
       return @source_url
     end
   end
 
+  def self.get_sha256 (architecture)
+    if !@build_from_source and @binary_sha256 and @binary_sha256.has_key?(architecture)
+      return @binary_sha256[architecture]
+    else
+      return @source_sha256
+    end
+  end
```

Comment:
This function appears to be an unnecessary abstraction since it's only called once and adds complexity.

Reply:
It makes things more readable, chromebrew is quite the wreck right now 😂

Reply:
If you're really sure you want it removed, I can revert 2f7c3d0

Reply:
If we're going to be sticklers for following best practices then we shouldn't be using `@` variables in a function since they act as global instead of local variables.  Is there anything wrong with the current logic?  As a general practice, I don't add a function unless it is necessary to call it multiple times.  Really the only thing that is a wreck is the dependency resolver algorithm.  Most everything else has already been optimized.

Reply:
Not a fun of putting this logic in a function.

Reply:
where would you put it, the way I have it the code is much more readable. Chromebrew, although it may be optimized, it's just not very readable at all.

Reply:
Now that I thought about it a bit, it does makes sense to have this function.  It is similar to the get_url function above.